content/en/docs/setup/independent/high-availability.md

```diff
@@ -139,11 +139,15 @@ different configuration.
 
 ### Bootstrap the first stacked control plane node
 
+{{< note >}}
+**Note**: Optionally replace `stable` with a different version of kubernetes such as `v1.12.0-beta.2` or `v1.12.0`.
+{{< /note >}}
+
 1.  Create a `kubeadm-config.yaml` template file:
 
         apiVersion: kubeadm.k8s.io/v1alpha2
         kind: MasterConfiguration
-        kubernetesVersion: v1.11.x
+        kubernetesVersion: stable
```

Comment:
shouldn't this alpha3?  

Also I'd just put the v1.12.0 in there explicitly and remove the meta comment.

Reply:
yeah, this is still WIP. rc1 is out so i'll finish this off today.

Reply:
please see above discussions regarding exact version vs stable & meta comment. There will have to be a comment regardless, but this is the easier path imo.